src/components/Asset/AssetActions/Download.tsx

```diff
@@ -183,26 +187,36 @@ export default function Download({
   const AssetAction = ({ asset }: { asset: AssetExtended }) => {
     return (
       <div>
-        {isUnsupportedPricing ? (
+        {isTemporaryDisabled ? (
           <Alert
             className={styles.fieldWarning}
             state="info"
-            text={`No pricing schema available for this asset.`}
+            text={`The publisher temporary disabled ordering for this asset`}
```

Comment:
Typo, same for variable name:

```suggestion
            text={`The publisher temporarily disabled ordering for this asset`}
```

Maybe call the variable 'isOrderDisabled' or something?

Reply:
the suggestion is very confusing for me. Sure i can change the variable to `isOrderDisabled`. Also the text is ok
![image](https://user-images.githubusercontent.com/26000280/196678048-31f35dc8-bb21-4989-8146-990d70b0e2ca.png)

Reply:
yes, but in this context you are looking for an adverb, not an adjective as you are not modifying a noun. So with adjective it would only work like so, but it sounds weird: "Temporary disability of ordering by the publisher."

As "disabled" is not a noun we have to use an adverb.

Reply:
touché